test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -631,6 +631,10 @@ public boolean checkSucceededJobStatus(String jobName) {
         return checkSucceededJobStatus(getNamespace(), jobName, 1);
```

Comment:
Guess we should remove calls like this because it's not reliable from my pov.

Reply:
Yes, we do but I think it would be a big change (https://github.com/strimzi/strimzi-kafka-operator/issues/6597)